### PR TITLE
PWN-9446 - fix formatter + tests

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9446

## Description of Work

PWN-9446 - fix formatter + tests

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapper.kt [hunk 1]
```diff
@@ -15,8 +14,7 @@ class BalanceMapper() {
 
     fun mapBalanceForWallet(balance: BigDecimal): String {
         return when {
-            balance.lessThenMinValue() -> BigDecimal.ZERO.asUsd()
-            balance < thousand -> balance.asUsd()
+            balance < thousand -> "$${balance.setScale(0, RoundingMode.FLOOR).stripTrailingZeros().toPlainString()}"
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapper.kt:17]: can you extract the big balance.setScale(0, RoundingMode.FLOOR).stripTrailingZeros().toPlainString() to a val? it's a big string template

File: app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import org.junit.Test
+import java.math.BigDecimal
+import org.p2p.wallet.utils.assertThat
+
+class BalanceMapperTest {
+    @Test
+    fun `GIVEN different data TO check correct cases correct for $0`() {
+        val zero = "$0"
+        val mapper = BalanceMapper()
+        val smallerThenOne1 = BigDecimal(0.001)
```
Comment (gslevinkov) [inline app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt:13]: can be a list and list can be mapped and asserted

File: app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package org.p2p.wallet.home.ui.container.mapper
+
+import assertk.assertions.isEqualTo
+import org.junit.Test
+import java.math.BigDecimal
+import org.p2p.wallet.utils.assertThat
+
+class BalanceMapperTest {
+    @Test
+    fun `GIVEN different data TO check correct cases correct for $0`() {
```
Comment (gslevinkov) [inline app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt:10]: GIVEN smaller than 1 balances WHEN map balance for wallet THAN balance is 0

File: app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        val ninety = BigDecimal(90.20)
+        val hundred = BigDecimal(100.1341)
+        mapper.mapBalanceForWallet(oneDollar).assertThat().isEqualTo("$1")
+        mapper.mapBalanceForWallet(tenFifty).assertThat().isEqualTo("$10")
+        mapper.mapBalanceForWallet(ninety).assertThat().isEqualTo("$90")
+        mapper.mapBalanceForWallet(hundred).assertThat().isEqualTo("$100")
+    }
+
+    @Test
+    fun `GIVEN different data TO check correct cases correct for max`() {
```
Comment (gslevinkov) [inline app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt:43]: GIVEN balance WHEN map balance THEN get MAX balance string

**Review by gslevinkov — APPROVED** [review]

File: app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+import assertk.assertions.isEqualTo
+import org.junit.Test
+import java.math.BigDecimal
+import org.p2p.wallet.utils.assertThat
+
+class BalanceMapperTest {
+    @Test
+    fun `GIVEN different data TO check correct cases correct for $0`() {
+        val zero = "$0"
+        val mapper = BalanceMapper()
```
Comment (gslevinkov) [inline app/src/test/java/org/p2p/wallet/home/ui/container/mapper/BalanceMapperTest.kt:12]: BalanceMapper -> WalletBalanceMapper

**Review by gslevinkov — APPROVED** [review]

**Review by eduardmaximovich — APPROVED** [review]

**Pull request merged**